tc/lang/sema.h

```diff
@@ -188,23 +191,27 @@ struct Sema {
     }
     return matched_type;
   }
+
```

Comment:
I was repeatedly getting confused about where a function started and ended. So I added these white lines to help. is this okay @zdevito ? :)